README.md

```diff
@@ -1,40 +1,123 @@
-$ npm install -g grunt-cli
 ```
+python manage.py runserver
+```
+
+### Data
 
-Then initialize the database, build the front-end, and run the server:
+For an instance of this pilot to display notice data, it needs to have access
+to said data. There are two basic schemes to set this up – you can either
+point your checkout to existing parsed data (ideal for folks only working on
+the front-end or who want to see a UI quickly), or you can populate your
+database by parsing them yourself.
 
-```bash
-$ python manage.py migrate --fake-initial
-$ python manage.py compile_frontend
-$ python manage.py runserver
+#### Point to Existing Data
+In this scenario, we just need to configure the UI to point to the live API:
+
+```
+echo "API_BASE = 'https://epa-notice.usa.gov/api/'" >> local_settings.py
+```
+
+As present, all of the data is behind a Basic Auth wall. You can find the
+relevant username and password from cloud.gov:
+
+```
+cf target -o eregsnc -s prod
+cf env eregs-web | grep HTTP_AUTH_USER
+cf env eregs-web | grep HTTP_AUTH_PASSWORD
 ```
 
+Then add that info to your `local_settings.py`:
+
+```
+echo "API_BASE = 'https://[AUTH_USER]:[AUTH_PASS]@epa-notice.usa.gov/api/'" >> local_settings.py
+```
+
+#### Parsing
+To parse the relevant data, you will need to also install the parsing library.
+
+```
+pip install git+https://github.com/eregs/regulations-parser.git
+```
+
+Then, you will want to start your local server and send it the parsed data.
+These steps will take several minutes. You can find the contents of
+"epa-rule.xml" in the shared Google Drive (it is sensitive).
+
+```
+python manage.py runserver &    # start the server as a background process
+eregs proposal_pipeline /path/to/epa-rule.xml http://localhost:8000/api --only-latest
+```
+Then navigate to http://localhost:8000
+
+### Editable Libraries
+Though this repository contains all of the pilot-specific code, you will most
+likely want to extend functionality in the base libraries as well.  To do
+this, fork and check out the appropriate library (regulations-site,
+regulations-core, regulations-parser) into a separate directory, then install
+it via
+
+```
+pip install -e /path/to/that/checkout
+```
+
+This will tell Python to use your local version of that library rather than
+the upstream version. Although modifying Python will trigger the `manage.py`
+command to restart, template changes will not. Modifying LESS or JS will also
+require re-running `compile_frontend`.
+
+### Fixing Errors
+If you pulled down the latest code changes, ran the server locally, and
+something didn’t build correctly (you got an error message instead of seeing
+eRegs), try running standard updates:
+
+Run `pip install -r requirements.txt` to pull in the latest dependencies. Note
+that this resets state from the _Editable Libraries_ technique above.
+
+If working with the parser, do `eregs clear`.
+
+Or if working with notice-and-comment, do `python manage.py compile_frontend`.
```

Comment:
Maybe around line 31 instead? We probably don't want to shortcut anything when trying to find an error.

Would you mind writing a quick sentence or two explaining it?